config.py

```diff
@@ -2,6 +2,28 @@
 from datetime import timedelta
 
 
+def get_mock_email_config() -> bool:
+    MOCK_EMAIL = os.getenv("MOCK_EMAIL")
+
+    #if MOCK_EMAIL env variable is set
+    if  MOCK_EMAIL: 
+        # MOCK_EMAIL is case insensitive
+        MOCK_EMAIL = MOCK_EMAIL.lower()
+        
+        if MOCK_EMAIL=="true":
```

Comment:
Why are these string values?

Reply:
@ramitsawhney27  Yes, Ideally we should have boolean. But the package which we use to manage environment variables `python-dotenv==0.8.2` doesn't have boolean support. For more details please visit https://dev.to/nicolaerario/comment/fe1e.

Reply:
Thank you for explaining it @naveennvrgup . I tested this now and it works as expected. Since @ramitsawhney27 did not request change I am assuming this is brings room for improvement. If you find a way this can be improved, please let us know or create an issue ;)